deploy/docker-compose.yml

```diff
@@ -95,10 +100,17 @@ volumes:
   datameta-filestorage:
     external: true
   certs:
+    external: true
   html:
+    external: true
   vhost:
+    external: true
   dhparam:
+    external: true
 
+# Setting external to false for networks means Docker will look for an existing network rather than making a throwaway
+# one when you run `docker-compose up [args]`. Run `docker network create <network name>` to create the network if it
+# doesn't already exist on the local docker engine.
 networks:
   net:
     external: false
```

Comment:
Do we have to use an external network? Why won't a network created by docker work here?

Reply:
This is not a necessary thing to do so long as this container is an isolated instance that doesn't need to communicate with other Docker services outside of the networks named in compose. I kept the networks as external to allow that possibility.